.travis.yml

```diff
@@ -18,17 +18,18 @@ env: JRUBY_OPTS="$JRUBY_OPTS --debug"
 rvm:
   # Include JRuby first because it takes the longest
   - jruby-9.1.13.0
-  - 2.2
-  - 2.3.4
-  - 2.4.1
+  - 2.2.9
```

Comment:
Can you please (while you're at it) change those to be `2.2`, `2.3`, etc so that travis will pick latest automatically

Reply:
Yes, done!